api/swagger-spec/v1.json

```diff
@@ -13235,7 +13235,7 @@
    },
    "v1.EmptyDirVolumeSource": {
     "id": "v1.EmptyDirVolumeSource",
-    "description": "EmptyDirVolumeSource is temporary directory that shares a pod's lifetime.",
+    "description": "EmptyDirVolumeSource represents an empty directory for a pod. Empty directory volumes support ownership management and SELinux relabeling.",
```

Comment:
I'll add another note about that in.  'Temporary' isn't quite right -- it's scoped to the lifetime of the pod.  There's been some confusion about this in the community and we likely need to improve the docs too.